src/distributions/uniform.rs

```diff
@@ -147,19 +148,29 @@ use serde::{Serialize, Deserialize};
 /// ```
 /// use rand::distributions::{Distribution, Uniform};
 ///
-/// fn main() {
-///     let between = Uniform::from(10..10000);
-///     let mut rng = rand::thread_rng();
-///     let mut sum = 0;
-///     for _ in 0..1000 {
-///         sum += between.sample(&mut rng);
-///     }
-///     println!("{}", sum);
+/// let between = Uniform::from(10..10000);
+/// let mut rng = rand::thread_rng();
+/// let mut sum = 0;
+/// for _ in 0..1000 {
+///     sum += between.sample(&mut rng);
 /// }
+/// println!("{}", sum);
+/// ```
+///
+/// If only one sample from the range is required, [`Rng::gen_range`] can be
+/// more efficient than sampling a `Uniform` distribution, but this is less
+/// efficient if multiple samples are taken.
```

Comment:
Combine with second para. of doc comment?

Reply:
Makes sense!